ext/liquid_c/block.c

```diff
@@ -308,72 +269,72 @@ static VALUE block_body_remove_blank_strings(VALUE self)
     }
     ensure_not_parsing(body);
 
-    size_t *const_ptr = (size_t *)body->constants.data;
-    const uint8_t *ip = body->instructions.data;
+    size_t *const_ptr = (size_t *)body->code.constants.data;
+    const uint8_t *ip = body->code.instructions.data;
 
-    while (true) {
-        switch (*ip++) {
-            case OP_LEAVE:
-                goto loop_break;
-            case OP_WRITE_RAW:
-            {
-                const_ptr++;
-                size_t *size_ptr = const_ptr++;
-                if (*size_ptr) {
-                    *size_ptr = 0; // effectively a no-op
-                    body->render_score--;
-                }
-                break;
+    while (*ip != OP_LEAVE) {
+        if (*ip == OP_WRITE_RAW) {
+            size_t *size_ptr = &const_ptr[1];
+            if (*size_ptr) {
+                *size_ptr = 0; // effectively a no-op
```

Comment:
Yes, although it would then have an unused constant arguments.  I imagine that a `OP_NOP` would be faster, but I'm not concerned about further optimizing this edge case.  Primarily these blank strings are handled in this way to minimize impact on code that isn't relying on this feature while keeping backwards compatibility, so this seemed simpler than a no-op instruction.

Reply:
Regarding backwards compatibility, I was referring to removing/ignoring blank strings from blank blocks.  I think our only remaining dependency on nodelist is for tags.